### PR TITLE
initial airdrop of documentation

### PR DESCRIPTION
**Pull request by pmatulis**

### CODE REVIEW

**Comment by pmatulis** [comment]
@pmahnke Hi. This is ready to be reviewed and merged. I will ask some OpenStack people for review but is there a reason you placed a blocked label on this PR?

File: docs/clustering.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+microstack.launch cirros --name test --availability-zone nova:node-urey
+```
+
+The output will provide the instance's floating IP address (here 10.20.20.157).
+
+So to SSH to the instance:
+
+```bash
+ssh -i ~/.ssh/id_microstack cirros@10.20.20.157
+```
```
Comment (petevg) [inline docs/clustering.md]: You should note that, without additional network configuration, you need to ssh from a session on the control node.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Installation
+
+Begin by installing MicroStack via a [snap][microstack-snap]. Here we use the
+default `--stable` channel but, as with any snap, channels `--beta` and
+`--edge` are also available:
+
+```bash
+sudo snap install microstack --stable --classic
+```
```
Comment (petevg) [inline docs/index.md]: This should be --beta.

File: docs/pro-tips.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## Accessing Horizon on a remote server
+
+If you've installed MicroStack on a remote server you can use SSH local port
+forwarding to access Horizon:
+
+```bash
+sudo ssh -i <ssh-key> -N -L 8001:10.20.20.1:80 <user>@<server-ip>
+```
+
+Then point your browser at: `http://localhost:8001`.
```
Comment (petevg) [inline docs/pro-tips.md]: I've got a PR underway to open up horizon by default. This language is fine for now, though.

**Comment by pmahnke** [comment]
@pmatulis i put blocked on it because you have no instructions in the PR on how to review it, it has no labels asking for reviews, and it has a strange title that suggests it is a work in progress and you haven't communicated that you want it reviewed and pushed live, so until any actualy form of communication happens, I don't want my guys to just push it live by mistake.  You used to work on the web team, so I know you know how we work.

**Comment by pmahnke** [comment]
@petevg +1, or needs updates?

**Pull request merged**